app/pages/home-not-logged-in/research.jsx

```diff
@@ -41,6 +41,7 @@ counterpart.registerTranslations('en', {
 const GZ123_COUNT = 98989226
 const OUROBOROS_COUNT = 142800311
 const OTHERS_COUNT = 8680290
+const OUROBOROS_USER_COUNT = 124921
```

Comment:
Thanks for naming this well :)

Pity, past us didn't name the other counts with the context of what they are counting :(